tests/acceptance/test_organization_events_v2.py

```diff
@@ -172,3 +326,74 @@ def test_event_detail_view_from_errors_view(self, mock_now):
 
             assert self.browser.element_exists_by_test_id("older-event")
             assert self.browser.element_exists_by_test_id("newer-event")
+
+    def test_transactions_query_empty_state(self):
+        with self.feature(FEATURE_NAMES):
+            self.browser.get(self.path + "?" + transactions_query)
+            self.wait_until_loaded()
+            self.browser.snapshot("events-v2 - transactions query - empty state")
+
+    @patch("django.utils.timezone.now")
+    def test_transactions_query(self, mock_now):
+        mock_now.return_value = before_now().replace(tzinfo=pytz.utc)
+
+        event_data = generate_transaction()
+
+        self.store_event(data=event_data, project_id=self.project.id, assert_no_errors=True)
+
+        with self.feature(FEATURE_NAMES):
+            self.browser.get(self.path + "?" + transactions_query)
+            self.wait_until_loaded()
+            self.browser.snapshot("events-v2 - transactions query - list")
+
+    @patch("django.utils.timezone.now")
+    def test_modal_from_transactions_query(self, mock_now):
+        mock_now.return_value = before_now().replace(tzinfo=pytz.utc)
+
+        event_data = generate_transaction()
+
+        event = self.store_event(data=event_data, project_id=self.project.id, assert_no_errors=True)
+
+        with self.feature(FEATURE_NAMES):
+            # Get the list page
+            self.browser.get(self.path + "?" + transactions_query)
+            self.wait_until_loaded()
+
+            # Click the event link to open the modal
+            self.browser.element('[aria-label="{}"]'.format(event.title)).click()
+            self.wait_until_loaded()
+
+            self.browser.snapshot("events-v2 - transactions query - modal")
+
+    # TODO: deferred
+    # @patch("django.utils.timezone.now")
```

Comment:
I'm going to complete this in a follow up PR.

Reply:
Instead of commenting the whole test out you can use `@pytest.mark.skip(reason="not done")`